README.md

```diff
@@ -1,20 +1,5 @@
 # Preparing LaTex articles using GitHub for the Living Journal of Computational Molecular Sciences ([LiveCoMS](http://www.livecomsjournal.org/))
 
-## About the journal
```

Comment:
Seems like we should be replacing this with some kind of info on what this repository is about, e.g. `## This repository provides templates for LiveCoMS articles`, then under it a brief explanation of what to do with the repository (fork it!), perhaps also with a link to an explanation of the journal model.

Remember, this README.md is the MAIN THING people see when they come to github.com/livecomsjournal so it will help if we can tell them what they are supposed to find here and what they should do with it.

Reply:
(For example, THIS displays on the main page but it contains no link to the `information_for_authors.md` file, which has the actual instructions.)

I know we discussed this before, but maybe it is better to just put the `information_for_authors.md` contents in THIS file and have no separate file? 

Or, maybe what we said we wanted to do was:
1) Make THIS `README.md` be just the info about (a) what our templates repository is for and (b) what authors should do with it (so including what's currently in `information_for_authors.md`) -- that is, stuff which should get wiped out after they fork
2) Make `information_for_authors.md` contain everything they will STILL need after they fork
I'm forgetting.

If we did that, then we would make sure both this `README.md` and the `your.README.md` both link to the `information_for_authors.md` so the organization of the repo is clear both before AND after they follow our instructions to set up their paper.

Main point: Right now the organization of the repo is not clear before OR after forking -- readers are not directed to `information_for_authors.md` anywhere, but they have to get there to find the instructions they need to follow to fork. Who will tell them to look there?